resources/istio-kyma-patch/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+name: istio-kyma-patch
+version: 1.0.0
+appVersion: 1.0.0
+tillerVersion: ">=2.7.2-0"
+description: Helm chart for istio patch required by kyma
+keywords:
+  - istio
+  - kyma
```

Comment:
We should add README here with description and point to components/istio-kyma-patch/README for the details